packages/eth2-types/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@chainsafe/eth2-types",
+  "version": "0.1.0",
+  "description": "> TODO: description",
+  "main": "index.js",
+  "scripts": {
+    "check-types": "tsc --noEmit --incremental",
+    "test": "nyc --cache-dir .nyc_output/.cache -r lcov -e .ts mocha -r ts-node/register 'test/**/*.test.ts' && nyc report"
+  },
+  "keywords": [],
+  "author": "Gregory Markou <gregorymarkou@gmail.com>",
```

Comment:
chainsafe? haha

Reply:
boiler plate lol, ill get to this eventually